source/help/settings/account-settings.md

```diff
@@ -14,6 +14,9 @@ Usernames are unique identifers appearing next to all posts. Pick something easy
 #### Nickname
 Nicknames appear in the Direct Messages member list and team management modal. You will not receive mention notifications when someone types your nickname unless you add it to the *Words That Trigger Mentions* in **Account Settings > Notifications**.
 
+#### Position
+Position can be used to describe your role or job title. It appears in the profile popup that shows up when you click a user's name in the center channel.
```

Comment:
Hmm yeah, I actually considered adding that but wasn't sure what our current stance was on clarity over completeness. 

In this case I'll add it, it doesn't matter too much here, but I think there's a lot of places in our docs where we focus on completeness over clarity/usefulness to the end user.